packages/x-components/src/views/home/Home.vue

```diff
@@ -189,6 +189,21 @@
             <!-- Facets -->
             <Facets class="x-list--gap-06" renderable-facets="!rootCategories_facet">
               <!--  Hierarchical Facet    -->
+              <template #category-paths="{ facet }">
+                <BaseHeaderTogglePanel class="x-facet">
+                  <template #header-content>
+                    <span class="x-ellipsis">{{ facet.label }}</span>
+                    <ChevronDown />
+                  </template>
+                  <!-- Filters -->
+                  <SlicedFilters max="4" :filters="facet.filters">
+                    <FiltersList v-slot="{ filter }">
+                      <HierarchicalFilter :filter="filter" :data-test="`${facet.label}-filter`" />
+                    </FiltersList>
+                  </SlicedFilters>
+                </BaseHeaderTogglePanel>
+              </template>
+
               <template #hierarchical-category="{ facet }">
```

Comment:
Should we remove the hierarchical one?

Reply:
We didn't remove it because the mocked responses for the `e2e` tests are using `hierarchical-category` as the `facet`. We have to refactor those too if we want to remove it. We didn't want to do it in this pull request because it would be way bigger than it already is.

Reply:
do we created a task for that?